viewer/src/utilities/networking/types.ts

```diff
@@ -0,0 +1,27 @@
+/*!
+ * Copyright 2020 Cognite AS
+ */
+
+import { File3dFormat } from '../File3dFormat';
+
+import { IdEither } from '@cognite/sdk';
+
+export interface BlobOutputMetadata {
+  blobId: number;
+  format: File3dFormat | string;
+  version: number;
+}
+
+export interface ModelUrlProvider<TModelIdentifier> {
+  getModelUrl(params: TModelIdentifier): Promise<string>;
+}
+
+export interface ByUrlModelIdentifier {
+  discriminator: 'external';
+  url: string;
+}
+
+export interface CdfModelIdentifier {
+  discriminator: 'cdf';
```

Comment:
Again, I think these can be without a discriminator

Reply:
Again, just moved stuff. I didn't add the discriminator - you did.

Reply:
I think the problem here is that in my commit I deleted some files, and when I merged these files had been moved, and been restored. A lot of the files deleted in this PR were files I had removed. I will go over this PR and fix it.